OHHTTPStubs/Sources/OHHTTPStubs.m

```diff
@@ -433,13 +433,12 @@ - (void)startLoading
                         case 308:
                             //Preserve the original request method and set the new location URL
                             [mReq setHTTPMethod:self.request.HTTPMethod];
-                            [mReq setURL:redirectLocationURL];
                             break;
 
                         default:
                             break;
                     }
-
+                    [mReq setURL:redirectLocationURL];
```

Comment:
303 shouldn't make a copy of the original request, as the intent according to RFC is to tell that the first request has been processed correctly and that we should now continue to a _new_ request